playwright-e2e/pages/mixins/hearing-details-mixin.ts

```diff
@@ -0,0 +1,33 @@
+import {Page} from "@playwright/test";
+
+export function HearingDetailsMixin(BasePage) {
+  return class extends BasePage {
+
+    constructor(page: Page) {
+      super(page);
+    }
+
+    async completeHearingDetails() {
+      await this.page.getByLabel('Case management', { exact: true }).check();
+      await this.page.locator('#hearingVenue').selectOption({ label: 'Swansea Crown Court' });
+      if (!(await this.page.getByLabel('In person').isChecked())) {
+        await this.page.getByLabel('In person').check();
+      }
+      await this.page.getByRole('textbox', { name: 'Day' }).click();
+      await this.page.getByRole('textbox', { name: 'Day' }).fill('1');
+      await this.page.getByRole('textbox', { name: 'Month' }).fill('6');
+      await this.page.getByRole('textbox', { name: 'Day' }).press('Tab');
+      await this.page.getByRole('textbox', { name: 'Year' }).fill('2');
```

Comment:
Focus on the element moved back to day and then to year. can be day,month,year. No need to click text box and then fill the text box.